hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,68 @@
+# Accounts Blocklisting
+
+## Purpose
+
+In Ethereum, everyone that knows a private key is able to authorize a transaction from the corresponding EVM public address, and the first to send a transaction would be eligible to withdraw any funds from that address.
+In Hedera, it is possible for an account with associated EVM address alias to have a key that does not correspond to the alias. This is what we refer to as “EVM address squatting” i.e. using a key that’s different from the key that’s used to derive the EVM address alias from.
+
+It is possible to “squat” on an EVM address in a couple of ways:
+1. Create an account with an alias and then update the account key.
+2. Create an account with an alias and a key that’s different from the key that’s used to derive the alias from.
+
+In both cases, this results in the private key losing control over the EVM address derived from the corresponding public key.
+
+This means that there is a difference in how some situations, like accidental transfers to known EVM addresses, will be handled in Ethereum and in Hedera:
+- In Ethereum, everyone that knows the corresponding private key will be able to authorize a transaction from the known address, and the first to send a transaction would be eligible to withdraw the funds.
+- In Hedera, only the account with the associated corresponding alias would be eligible to withdraw the funds. If another key controls this account, then only that key can be used to withdraw the funds.
```

Comment:
Ah! This is pretty strong.  Stronger than I suggested above.  In this case not only is the Hedera account key holder able to drain the account but the Ethereum private key holder is _unable_ to.  Correct?

Reply:
correct. that's how Hedera differs in behavior, since in Ethereum it's not _possible_ to change the controlling key of an acocunt, it's always the key used to derive the EVM address from.